server/parameters/CoreSubcommandParser.java

```diff
@@ -99,27 +100,27 @@ public List<Help> helpList() {
         }
     }
 
-    public static class Import extends SubcommandParser<ServerSubcommand.Import> {
+    public static class Import extends SubcommandParser<CoreSubcommand.Import> {
 
         public static final String[] tokens = new String[]{"import"};
         public static final String description = "Run TypeDB import.";
 
-        private static final OptionParser.String database =
+        public static final OptionParser.String database =
```

Comment:
same as above - allow referencing from within the equivalent Cluster implementation